src/content/docs/en/guides/integrations-guide/mdx.mdx

```diff
@@ -84,13 +84,136 @@ For other editors, use the [MDX language server](https://github.com/mdx-js/mdx-a
 
 ## Usage
 
-With the Astro MDX integration, you can [add MDX pages to your project](/en/guides/markdown-content/#markdown-and-mdx-pages) by adding `.mdx` files within your `src/pages/` directory. You can also [import `.mdx` files](/en/guides/markdown-content/#importing-markdown) into `.astro` files.
+Visit the [MDX docs](https://mdxjs.com/docs/what-is-mdx/) to learn about using standard MDX features.
 
-Astro's MDX integration adds extra features to standard MDX, including Markdown-style frontmatter. This allows you to use most of Astro's built-in Markdown features like a [special frontmatter `layout` property](/en/guides/markdown-content/#frontmatter-layout).
+## MDX in Astro
 
-See how MDX works in Astro with examples in our [Markdown & MDX guide](/en/guides/markdown-content/).
+Adding the MDX integration enhances your Markdown authoring with JSX variables, expressions and components.
 
-Visit the [MDX docs](https://mdxjs.com/docs/what-is-mdx/) to learn about using standard MDX features.
+It also adds extra features to standard MDX, including support for Markdown-style frontmatter in MDX. This allows you to use most of [Astro's built-in Markdown features](/en/guides/markdown-content/).
+
+`.mdx` files must be written in [MDX syntax](https://mdxjs.com/docs/what-is-mdx/#mdx-syntax) rather than Astro’s HTML-like syntax.
+
+### Using Exported Variables in MDX
+
+MDX supports using `export` statements to add variables to your MDX content.
+
+For example, you can export a `title` field from an MDX page or component to use as a heading with `{JSX expressions}`:
+
+```mdx title="/src/pages/posts/post-1.mdx"
+export const title = 'My first MDX post'
+
+# {title}
+```
+
+MDX files can also use an `export` statement to export data to a component that imports it.
+
+For example, you can export a `title` field from an MDX page or component.
+
+```mdx title="/src/pages/posts/post-1.mdx"
+export const title = 'My first MDX post'
+```
+
+This `title` will be accessible from `import` and `import.meta.glob()` statements:
```

Comment:
When I first saw the two code blocks with `export const title` and "*For example, you can export a `title` field from an MDX page or component*" I thought you forgot to remove one. After rereading, I understood the intention but I wonder if it can't be simplified to avoid duplication:
````suggestion
MDX supports using `export` statements to add variables to your MDX content or to export data to a component that imports it.

For example, you can export a `title` field from an MDX page or component to use as a heading with `{JSX expressions}`:

```mdx title="/src/pages/posts/post-1.mdx"
export const title = 'My first MDX post'

# {title}
```

Or you can use that exported `title` in your page using `import` and `import.meta.glob()` statements:
````
But I'm used to MDX, so my vision may be biased... Your approach may be better for a beginner. I'll let you judge!

Reply:
Yes, you're right. This was copied in separate chunks from different places and I didn't smooth the transition very well!